jax/experimental/jax2tf/impl_no_xla.py

```diff
@@ -792,10 +792,99 @@ def _dynamic_update_slice(operand, update, *start_indices,
 
 tf_impl_no_xla[lax.dynamic_update_slice_p] = _dynamic_update_slice
 
-tf_impl_no_xla[lax.scatter_p] = _unimplemented("scatter")
-tf_impl_no_xla[lax.scatter_min_p] = _unimplemented("scatter min")
-tf_impl_no_xla[lax.scatter_max_p] = _unimplemented("scatter max")
-tf_impl_no_xla[lax.scatter_mul_p] = _unimplemented("scatter mul")
-tf_impl_no_xla[lax.scatter_add_p] = _unimplemented("scatter add")
+def get_move_to_leading_axis_bijector(axis):
+  '''
+  Bijector taking an array x, of rank `operand_rank`.
+  Forward shift the axis `axis` to the leading axis
+
+  so for array `operand_rank`=4 and `axis=4`. The last axis is
+  moved for the front: permutation - [3,0,1,2]
```

Comment:
This docstring is a bit confusing, because this function actually returns two functions, which is not really clear. Also the name `get_move_to_leading_axis_bijector` is maybe not very enlightening.

One option is to just define a function `shift_axis_forward(operation, axis, inverse=False)` and define `_fwd = partial(shift_axis, axis=axis)` and `_bwd = partial(_forward, inverse=True)` when you need it below (you only use them once anyway).

Reply:
Agreed, it's a terrible name. Will alter.